src/main/java/lotto/domain/LottoStatistics.java

```diff
@@ -0,0 +1,46 @@
+package lotto.domain;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import static lotto.domain.LottoInformation.*;
+
+public class LottoStatistics {
+
+    private final Map<Integer,  Integer> statisticsMap;
+
+    public LottoStatistics(LottoTickets lottoTickets, List<Integer> winNumbers) {
```

Comment:
```suggestion
    public LottoStatistics(LottoTickets lottoTickets, LottoTicket winLotto) {
```

winNumbers 도 하나의 로또티켓으로 볼 수 있을 것 같아요 😄